website/docs/r/rds_instance_state.html.markdown

```diff
@@ -0,0 +1,59 @@
+---
+subcategory: "RDS (Relational Database)"
+layout: "aws"
+page_title: "AWS: aws_rds_instance_state"
+description: |-
+  Terraform resource for managing an AWS RDS (Relational Database) RDS Instance State.
+---
+
+# Resource: aws_rds_instance_state
+
+Terraform resource for managing an AWS RDS (Relational Database) RDS Instance State.
+
```

Comment:
Clarifying the destroy behavior might be worthwhile here, just to set expectations that state won't change as part of a destroy operation.

```suggestion

~> Destruction of this resource is a no-op and **will not** modify the instance state

```